Package.swift

```diff
@@ -7,9 +7,9 @@ let package = Package(
         .Package(url: "https://github.com/vapor-community/postgresql.git", Version(2,0,0, prereleaseIdentifiers: ["beta"])),
```

Comment:
Go ahead and change the postgresql dependency to `majorVersion: 2` as well. We can merge this PR when PostgreSQL hits 2.0.